src/arch/host/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ target_include_directories(sof_public_headers INTERFACE ${PROJECT_SOURCE_DIR}/sr
 target_include_directories(sof_public_headers INTERFACE ${PROJECT_SOURCE_DIR}/src/platform/library/include)
 
 # C & ASM flags
-target_compile_options(sof_options INTERFACE -g -O3 -Wall -Werror -Wl,-EL -Wmissing-prototypes -Wimplicit-fallthrough=3 -Wpointer-arith -DCONFIG_LIBRARY)
+target_compile_options(sof_options INTERFACE -g -O3 -Wall -Werror -Wl,-EL -Wmissing-prototypes
+  -Wimplicit-fallthrough=3 -Wpointer-arith -DCONFIG_LIBRARY -imacros "${CONFIG_H_PATH}")
```

Comment:
For some strange reason github has decided authors cannot request changes on their own PRs but consider I did and please don't merge this until I change this line and the other one below to remove the space after the `-imacros` option. Long story in https://github.com/zephyrproject-rtos/zephyr/commit/e53c0d0ec7640bd2, thanks @jajanusz for pushing me to do that research.